.github/ariane-config.yaml

```diff
@@ -32,9 +32,15 @@ triggers:
   /ci-external-workloads:
     workflows:
     - conformance-externalworkloads.yaml
+  /ci-gateway-api:
+    workflows:
+    - conformance-gateway-api.yaml
```

Comment:
OK so now we have to add `ci-gateway-api` to the name in `.github/workflows/conformance-gateway-api.yaml` xD

Reply:
Aaaah but it's missing on `main`: https://github.com/cilium/cilium/blob/main/.github/workflows/conformance-gateway-api.yaml

So disregard I guess?

Reply:
I'd say disregard for now and fix/backport later as it will again need manual adjustment of required checks on all affected branches :disappointed: